torch/csrc/jit/passes/onnx.cpp

```diff
@@ -186,11 +187,40 @@ void BlockToONNX(
     return it->second;
   };
 
+  GRAPH_DEBUG("BlockToONNX: graph of old block: ", old_block->owningGraph()->toString());
+
   // Initialize context and environment
   for (auto input : old_block->inputs()) {
     auto n = ctx.block->addInput()->copyMetadata(input);
     env[input] = n;
   }
+
+  // Merge ONNX inferred type with existing type.
+  // If inferred type is sequence, overwrite old type because it is more compatible
+  // with ONNX shape inference.
+  // If inferred type is tensor type, merge the two types, with inferred type having
+  // higher precedence.
+  auto mergeOutputTypes = [](TypePtr old_type, TypePtr new_type) -> TypePtr {
```

Comment:
Minor comment about naming. We should stick to one convention for variable naming. For C++, the precedent in these files (which is also largely true in general) is to use camel case for variable names, without any underscores. So `old_type` can be `oldType`. For Python the most common convention is to use all lower case with underscores. 
Let's try to pick one convention for C++ and follow it.

Reply:
agree should stick to one convention. Currently it looks like both c++ and python files are using underscores, will keep the modifications to it.